README.DATUMGRID

```diff
@@ -1,14 +1,31 @@
-	US/Canada/France/New Zealand Datum Shifting Files
-	-------------------------------------------------
+Datum Shifting Files for PROJ
+-------------------------------------------------
 
-The files in this package can be unpacked in the PROJ.4 data directory. 
+The files in this package can be unpacked in the PROJ.4 data directory.
 For an installed PROJ.4 this may be /usr/local/share/proj or /usr/share/proj
 on unix style operating systems .
 
-Note that this package includes "CTable2" format grid shift files for 
+Note that this package includes "CTable2" format grid shift files for
 NAD27 in the USA and require PROJ 4.8.0 or newer.  On the plus side there
 is no need to compile ascii files into binary files with this version as
 CTable2 files are not system dependent (as distinct from the CTable files
-produced in PROJ 4.7.0 and older). 
+produced in PROJ 4.7.0 and older).
 
+About the Datum Grid package
+-------------------------------------------------
 
+The datum grid package for PROJ is maintained on the OSGeo proj-datumgrid
+repository on GitHub [0]. All grids in the package are released under
+permissives licenses. New grids are accepted into the package as long as they
+are released under a suitable license that allow redistribution and the source
```

Comment:
Perhaps we should explicitly state that the license needs to be compatible with the [Open Source Definition](https://opensource.org/osd) ([Annotated](https://opensource.org/osd-annotated))